proto/public/api.proto

```diff
@@ -66,9 +66,8 @@ message MetricSeries {
 }
 
 message MetricMetadata {
-  string targetPod = 1;
+  reserved 1, 3;
```

Comment:
Our histogram change probably already broke API compatibility so 0.2 -> 0.3 is going to be a breaking change.  Therefore, I don't think we need to reserve fields 1 and 3.  🍳